llvm/unittests/Analysis/VectorFunctionABITest.cpp

```diff
@@ -44,10 +46,11 @@ class VFABIParserTest : public ::testing::Test {
   FunctionCallee F;
 
 protected:
-  // Referencies to the parser output field.
+  // References to the parser output field.
   ElementCount &VF = Info.Shape.VF;
   VFISAKind &ISA = Info.ISA;
   SmallVector<VFParameter, 8> &Parameters = Info.Shape.Parameters;
+  size_t ScalarParametersNum;
```

Comment:
i think would be good to initialise it to 0 here

Reply:
I didn't as there's a [reset](https://github.com/llvm/llvm-project/pull/73907/files/0ad4a31a051930353b1168b18216ff570b43f5c3#diff-c28f41966e76f1c7cf2a1b7f1e77e486f2e9f91fa7731fa0b7fd1cbead6f84d3R38)  method that deals with all other uninitialized variables when `invokeParser` gets called. I could move it closer to those variables (to line [46](https://github.com/llvm/llvm-project/pull/73907/files/0ad4a31a051930353b1168b18216ff570b43f5c3#diff-c28f41966e76f1c7cf2a1b7f1e77e486f2e9f91fa7731fa0b7fd1cbead6f84d3R46)) but if you think it's still not clear I could just initialize it.